src/bindings/js/node/tests/unit/infer_request.test.js

```diff
@@ -4,59 +4,62 @@
 
 const { addon: ov } = require('../..');
 const assert = require('assert');
-const { describe, it, before } = require('node:test');
+const { describe, it, before, beforeEach } = require('node:test');
 const { testModels, isModelAvailable, getModelPath } = require('./utils.js');
 
 const epsilon = 0.5; // To avoid very small numbers
 const testXml = getModelPath().xml;
 
 describe('ov.InferRequest tests', () => {
 
+  let compiledModel = null;
+  let tensorData = null;
+  let tensor = null;
+  let resTensor = null;
+  let tensorLike = null;
+
   before(async () => {
     await isModelAvailable(testModels.testModelFP32);
+
+    const core = new ov.Core();
+    const model = core.readModelSync(testXml);
+    compiledModel = core.compileModelSync(model, 'CPU');
+
+    tensorData = Float32Array.from({ length: 3072 }, () => (Math.random() + epsilon));
+    tensor = new ov.Tensor(
+      ov.element.f32,
+      [1, 3, 32, 32],
+      tensorData,
+    );
+    resTensor = new ov.Tensor(
+      ov.element.f32,
+      [1, 10],
+      tensorData.slice(-10),
+    );
+    tensorLike = [[tensor],
+      [tensorData]];
```

Comment:
Does it work without extra brackets, like here: https://github.com/openvinotoolkit/openvino/pull/26393/files?diff=split&w=0#diff-d3c56e09549eb996bb3b0d9ab3cf8941524245e0cb1bcaead663d721a3e71619R27?

Reply:
Yeah, here you just have to be remember that infer() doesn't accept a single Tensor or TypedArray